src/views/Home.jsx

```diff
@@ -1,7 +1,41 @@
 import './Home.css';
 import { SingleList } from '../components/SingleList';
+import { useEffect, useState } from 'react';
+import { createList } from '../api/firebase';
+import { useNavigate } from 'react-router-dom';
+
+export function Home({ data, setListPath, user }) {
+	const [shoppingListName, setShoppingListName] = useState('');
+	const navigate = useNavigate();
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		const { uid, email } = user;
+
+		const names = data.map((list) => {
+			return list.name;
+		});
+
+		if (names.includes(shoppingListName)) {
+			alert('name should be unique');
+
+			return;
+		}
+
+		createList(uid, email, shoppingListName);
+		alert('created');
+		navigate('/list');
```

Comment:
Issue: when a new list is created, we are navigated to `/list` as expected, but the local storage is not updating as expected. You can see what is in your local storage by running `localStorage.getItem('tcl-shopping-list-path')` in your browser console.

The `useEffect` below doesn't seem to be running after a new list is created...might be worth checking that out.